presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -866,6 +866,36 @@ Examples:
 
     CALL iceberg.system.remove_orphan_files(schema => 'db', table_name => 'sample');
 
+Fast Forward Branch
+^^^^^^^^^^^^^^^^^^^
+
+This procedure allows you to advance the current branch to a more recent snapshot from another branch without replaying any intermediate snapshots.
+We can perform a fast-forward of from ``branch`` up to the ``to`` snapshot if from ``branch`` is an ancestor of ``to``.
```

Comment:
```suggestion
``branch`` can be fast-forwarded up to the ``to`` snapshot if ``branch`` is an ancestor of ``to``.
```
Revision suggested to emphasize that ``branch`` is being acted on, and I thought the `from` was distracting. Also to emphasize the action (fast-forward) dependent on the condition `if ...`.